sdk/cosmosdb/cosmos/src/ClientContext.ts

```diff
@@ -543,6 +543,55 @@ export class ClientContext {
     return this.globalEndpointManager.getReadEndpoint();
   }
 
+  public async bulk<T>({
+    body,
+    path,
+    resourceId,
+    partitionKeyRange,
+    options = {}
+  }: {
+    body: T;
+    path: string;
+    partitionKeyRange: string;
+    resourceId: string;
+    options?: RequestOptions;
+  }) {
+    try {
+      const request: RequestContext = {
+        globalEndpointManager: this.globalEndpointManager,
+        requestAgent: this.cosmosClientOptions.agent,
+        connectionPolicy: this.connectionPolicy,
+        method: HTTPMethod.post,
+        client: this,
+        operationType: OperationType.Batch,
+        path,
+        body,
+        resourceType: ResourceType.item,
+        resourceId,
+        plugins: this.cosmosClientOptions.plugins,
+        options
+      };
+
+      request.headers = await this.buildHeaders(request);
+      request.headers[Constants.HttpHeaders.IsBatchRequest] = "True";
+      request.headers[Constants.HttpHeaders.PartitionKeyRangeID] = partitionKeyRange;
+      request.headers[Constants.HttpHeaders.IsBatchAtomic] = false;
```

Comment:
Should this be "False"?

Reply:
I'm not actually sure, I was going off a spec doc from the beginning of this project, I believe it gets stringified to 'false' in the request

```
// mandatory

x-ms-cosmos-is-batch-request: True

 

// Optional according to use case:

x-ms-cosmos-batch-continue-on-error: True/False

x-ms-cosmos-batch-atomic: false/true
```

Reply:
This gets stringified to `"false"`. Headers are case insensitive by HTTP spec